html/cross-origin-embedder-policy/README.md

```diff
@@ -1 +1,52 @@
 See `../cross-origin-opener-policy/README.md`.
+
+## Reporting
+
+### Existing tests ###
+
+- cache-storage-reporting-dedicated-worker.https.html
+- cache-storage-reporting-document.https.html
+- cache-storage-reporting-service-worker.https.html
+- cache-storage-reporting-shared-worker.https.html
+- reporting-navigation.https.html
+- reporting-subresource-corp.https.html
+- reporting-to-endpoint.https.html
+- reporting-to-owner.https.html
+
+### Testing plan ###
+
+- test multiple values with COEPRO
+- test "relevant settings object" is the right one
```

Comment:
I checked [these tests](https://github.com/web-platform-tests/wpt/pull/21206) which exercise the 4 possible environment settings objects.

For COEP reporting, there are two algorithms (#embedder-policy-checks) -- one for navigations, one for running workers. For navigations, the environment settings object for the API that causes the navigation shouldn't have any effect on the COEP report's choice of environment settings object, since it grabs a particular document's relevant environment settings object. But we can still test it, with invoking location.assign in convoluted ways with multiple documents in play. For workers, call the `Worker` constructor in convoluted ways (grab the constructor from a different document).

Reply:
I found a spec bug here: https://github.com/whatwg/html/issues/6518